### ISSUE TITLE
8.11.4. New - Screen Name/Title: Settings - Reply-to Address - Insufficient Semantic Structure for List Context

### ISSUE THREAD

**Issue by jzbahrai** [cds-snc/notification-planning#2476]
Component: Add Reply-to Address
Observation: 
The explanatory text and bullet list (`<ul class="list list-bullet">`) are visually and contextually grouped with the button ("Add reply-to address"), but they are separated structurally in code. This may cause confusion for screen reader users about the context of the list

(Attachment: 8.11.4. Insufficient Semantic Structure for List Context.png)

Recommendation: 
`Use semantic landmarks (e.g., <section> with <h2>) or ARIA roles (<div role="region" aria-labelledby="...">) to group and label related content to improve navigation and context clarity.
`
